client/src/styles.css

```diff
@@ -60,6 +60,11 @@
   right: -20px !important;
 }
 
+.ant-layout-sider-children {
+  border-right: 2px solid var(--global-color-primary--light) !important;
+  overflow: auto;
```

Comment:
@wesleyboar Can you remind us how we should be resolving long app labels?
![Screenshot 2024-05-30 at 1 39 40 PM](https://github.com/DesignSafe-CI/portal/assets/20326896/65097bfe-4de6-469a-8801-e0a3f4493c70)

Reply:
Ah, nevermind. We're going with a `shortLabel` field in an app's `notes`

Reply:
Truncate is good for now. I don't what `shortLabel` is, but I trust you.

I'd say **either** the text should wrap **or**, at least, the text is available somewhere else.[^1]

[^1]: Both options are so the user has a way to see enough to distinguish two items that start with similar text.